cleanup.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+rm -rf ${REPORT_FOLDER}
```

Comment:
Adding this cleanup job deletes files owned by root which causes problems on runners that don't run as root.